python/lsst/obs/base/gen2to3/rootRepoConverter.py

```diff
@@ -119,16 +120,19 @@ def runRawIngest(self, pool=None):
             dataRefs = self.butler2.subset(self.task.config.rawDatasetType)
         dataPaths = getDataPaths(dataRefs)
         self.task.log.info("Ingesting raws from root %s into run %s.", self.root, self.task.raws.butler.run)
-        self._rawRefs.extend(self.task.raws.run(dataPaths, pool=pool))
+        if not self.task.dry_run:
+            self._rawRefs.extend(self.task.raws.run(dataPaths, pool=pool))
```

Comment:
I think I'd expect dry run to issue a log message saying something like "Skipping raw ingest for {dataPaths}".